doc/user_guide/index.rst

```diff
@@ -38,32 +33,85 @@
    -----------------------------------------------------------------------------
    Written by R. W. Ford and A. R. Porter, STFC Daresbury Lab
 
+.. only:: html
+
+    .. image:: psyclone_v1.0.png
+        :width: 75%
+        :align: center
+        :alt: PSyclone
+
+PSyclone User's Guide
+=====================
+
+
+.. only:: html
+
+    PSyclone is a code generation system developed to produce domain-specific
+    languages (DSLs) for finite element, finite volume and finite difference
```

Comment:
Sorry to be picky but PSyclone doesn't produce DSLs. Maybe "developed to support domain-specific..."

Reply:
Yeah 'support' is better